homeassistant/components/elkm1/alarm_control_panel.py

```diff
@@ -107,18 +113,19 @@ class ElkArea(ElkAttachedEntity, AlarmControlPanelEntity, RestoreEntity):
         | AlarmControlPanelEntityFeature.ARM_AWAY
         | AlarmControlPanelEntityFeature.ARM_NIGHT
     )
+    _element: Area
 
-    def __init__(self, element, elk, elk_data):
+    def __init__(self, element: Element, elk: Elk, elk_data: dict[str, Any]) -> None:
         """Initialize Area as Alarm Control Panel."""
         super().__init__(element, elk, elk_data)
         self._elk = elk
-        self._changed_by_keypad = None
-        self._changed_by_time = None
-        self._changed_by_id = None
-        self._changed_by = None
-        self._state = None
+        self._changed_by_keypad: str | None = None
+        self._changed_by_time: str | None = None
+        self._changed_by_id: int = -1
```

Comment:
Please make the `-1` a named constant

Reply:
I'm assuming this is to initialize the `int` and it will never be seen?

Reply:
Resolved by hardening the code below where this is used.